vshard.go

```diff
@@ -113,6 +113,68 @@ func (rs *Replicaset) bucketStat(ctx context.Context, bucketID uint64) (BucketSt
 	return *bsInfo, bsError
 }
 
+type ReplicasetCallOpts struct {
+	PoolMode pool.Mode
+	Timeout  time.Duration
+}
+
+func (rs *Replicaset) ReplicasetCallImpl(
```

Comment:
looks legit
please add a small comment to the method too
also the question is why this code is located in vshard.go, and not api.go, it seems to carry more semantic load for calling the API

Reply:
move to file replicaset.go